### DISCUSSION TITLE
docker-compose: using only postgres and ztnet containers. with zerotier installed on host

### DISCUSSION THREAD

**Discussion by erlangparasu** [sinamics/ztnet#542] [Q&A]
docker-compose: using only postgres and ztnet containers. with zerotier installed on host

**Comment by sinamics** [comment]
Comment out the zerotier section and the zerotier volume in docker-compose.yml.
Then goto admin => controller and set the server_ip:9993 and secret for your new controller.


![image](https://github.com/user-attachments/assets/3e074214-7ee1-4cd0-acb8-a0e4d4392c3c)
